docs/src/spec/items/contracts.md

```diff
@@ -45,23 +51,64 @@ contract GuestBook {
 }
 ```
 
-## The `__init__` function
+Multiple contracts can be compiled from a single `.fe` contract file.
+
+## pragma
+
+Other smart contract languages require a `pragma` statement at the beginning of a contract. These are used to specify which compiler version to use to compile the contract. In Fe, pragma statements are not strict requirements in every contract. They are used to enable developers to express that certain code is designed to be compiled with a specific compiler version such that non-matching compiler versions will reject it.
+
+Read more on [pragma](../statements/pragma.md)
+
+## State variables
+
+State variables are permanently stored in the contract storage on the blockchain. Styate variables should be declared inside the contract body but outside the scope of any individual contract function. 
+
+```fe
+pub contract Example {
+    _some_number: u256
```

Comment:
Fe doesn't encourage the pattern of prefixing state variables with `_`. Maybe it makes sense in Solidity but state access in Fe is already clearly distinct from local access (e.g. `self.foo` vs `foo`)

Reply:
```suggestion
    some_number: u256
```